### PR TITLE
Update readme to refer to the documentation page

### PR DESCRIPTION
**Pull request by cmajed**
To avoid to update this every time we add user build dynamic rooting feature to a new android version.

### CODE REVIEW

**Review by guillaume-roche — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Copyright (C) Genymobile
 
 This repository stores Genymotion Android release keys, and corresponding .mk that we include internally in our `device.mk`. Third party developpers can use the release keys to sign their own applications in order to grant them system privileges on Genymotion. 
 
-**Note:** only the Android 10 version of Genymotion Device Image (PaaS) starting from v13.1 rely on these keys. Our other Android versions rely on the `test-keys` from AOSP. This note will be updated when we migrate other versions to the keys stored in this repository.
+**Notes:** Please refer to our [Root Access documentation page](https://docs.genymotion.com/paas/10_Using_root_access/) for more details about versions where this feature is available. Our other Android versions rely on the `test-keys` from AOSP. This note will be updated when we migrate other versions to the keys stored in this repository.
```
Comment (gvercoutere-geny) [inline README.md]:
```suggestion
**Notes:** Please refer to our [Root Access documentation page](https://docs.genymotion.com/paas/10_Using_root_access/) for more details about versions for which this feature is available. Our other Android versions rely on the `test-keys` from AOSP. This note will be updated when we migrate other versions to the keys stored in this repository.
```

Comment (gvercoutere-geny) [reply README.md]: Not sure, but it sounds better.

Comment (cmajed) [reply README.md]: Personally, I don't have a preference between the two formulations. :)

Comment (c-quenin) [inline README.md]: Are we sure that the listing of the versions which support release builds will be added to the https://docs.genymotion.com/paas/10_Using_root_access/ page? I don't know if Jue started working on it

Comment (cmajed) [reply README.md]: Yep I already checked it with @jmaneyrol69 
We already have a PR for it https://github.com/Genymobile/genymotion-doc-paas/pull/51

Comment (jmaneyrol69) [reply README.md]: Speaking of which, I've added @c-quenin and @cmajed as reviewer of the doc PR.

**Review by c-quenin — APPROVED** [review]

**Review by jmaneyrol69 — APPROVED** [review]

**Review by gvercoutere-geny — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gvercoutere-geny) [reply README.md]: Let's @jmaneyrol69 decide!

Comment (jmaneyrol69) [reply README.md]: quite honestly, it's quite the same. Let's say "for which" which may be more accurate.

**Comment by gvercoutere-geny** [comment]
The link https://docs.genymotion.com/paas/10_Using_root_access/ may change.
Do not merge yet.

**Comment by c-quenin** [comment]
> The link https://docs.genymotion.com/paas/10_Using_root_access/ may change. Do not merge yet.

In the end, the link is the same, see latest commit on https://github.com/Genymobile/genymotion-doc-paas/pull/51
So we can merge as it is.

**Comment by cmajed** [comment]
since doc was updated we are now able to merge this

**Pull request merged**